src/release.py

```diff
@@ -238,7 +238,7 @@ def make_tag(
     input(f">>> Ready to push to remote {remote_name}? Ctrl-c to cancel")
     print("")
     print(">>> Pushing...")
-    subprocess.check_call(["git", "push", "--tags", remote_name, tag_name])
```

Comment:
Prepending `refs/tags/` is only required if the tag name alone is ambiguous, e.g. because there's a branch with the same name. It doesn't hurt to include it, though, so we might as well.